ptxboa_streamlit.py

```diff
@@ -101,7 +106,10 @@
     st.session_state[st.session_state["tab_key"]] = "Dashboard"
 
 sac.tabs(
-    [sac.TabsItem(label=i) for i in tabs],
+    [
+        sac.TabsItem(label=i, icon=tabs_icons[i] if i in tabs_icons.keys() else None)
+        for i in tabs
+    ],
```

Comment:
it is clearer to use the [dict.get() method](https://docs.python.org/3/library/stdtypes.html#dict.get) if you want to set a default for non existent keys.

```suggestion
    [sac.TabsItem(label=i, icon=tabs_icons.get(i, None)) for i in tabs],
```

Reply:
Nice! I did not know that one!